e2e/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "cross-env WORKBENCH_ENV=dev jest",
```

Comment:
Maybe comment somewhere on which values need to live in this file vs at the various levels in the CircleCI configs.